report/src/index.tex

```diff
@@ -18,7 +18,9 @@
 
 \include{introduction/intro.tex}
 \include{problem_analysis/problem_analysis.tex}
-\include{sprint_1/index.tex}
 
 \printbibliography[heading=bibintoc, title=Bibliography]
 \label{bib:mybiblio.bib}
+\appendix
+\renewcommand{\thesection}{\Alph{section}}
```

Comment:
what's this for? I don't think you ever use it?

Reply:
Yes, this is for appendix so they start with A. For instance A1 User Stories.